examples/hello_world/ios/Runner/AppDelegate.h

```diff
@@ -1,9 +1,5 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
```

Comment:
The headers were added since they live in the Flutter repo.  That's concerning, would it override copyright headers or other changes to this file that our users introduced?

Reply:
I'll look into this.

The migrate command doesn't touch files that the user added and are not part of the templates, but in files like this that are generated by the tool, it does indeed try to sync it to the generated version (which doesn't include copyright headers). It would be possible to do extra processing to ignore changes to comments.

In any case, the proposed changes are all displayed as a diff in `flutter migrate status` and developers can review if the changes are appropriate before applying. The staged changes also live in the `migrate_working_directory` and can be changed before applying at any point.

Reply:
Actually, the headers should be counted as user additions since the templates dont have them. I'll take a look at why the merge is removing them

Reply:
This particular case was falling back on a two way merge when a 3 way merge is appropriate, it's a bug.